rest/config_test.go

```diff
@@ -655,8 +656,8 @@ func TestSetupAndValidateDatabases(t *testing.T) {
 
 	sc = &ServerConfig{Databases: databases}
 	errs = sc.setupAndValidateDatabases()
-	assert.Len(t, errs, 1)
-	assert.Contains(t, errs[0].Error(), "invalid control character in URL")
+	assert.NotNil(t, errs)
+	assert.Contains(t, errs.Errors[0].Error(), "invalid control character in URL")
```

Comment:
length check first

Reply:
Sure, this is addressed in latest revision.